bash_completion

```diff
@@ -2614,6 +2612,17 @@ _complete_as_root()
     [[ $EUID -eq 0 || ${root_command-} ]]
 }
 
+# Complete on available commands, subject to `no_empty_cmd_completion`.
+#
```

Comment:
I think we can define the behavior of `@return`, though there are choices.

Reply:
```suggestion
# @return True (0) if one or more completions are generated, or otherwise False
# (1).  Note that it returns 1 even when the completion generation is canceled
# by `shopt -s no_empty_cmd_completion`.
#
```